src/style.css

```
@@ -1 +1,27 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.block {
+  justify-content: center;
+}
+
+.block__stars {
+  display: flex;
+}
+
+.block__stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+}
+
+.block__stars__star.active {
```

```suggestion
.stars--1 ... {
...
```
Consider using the class with modifier to get active stars